wazp/utils.py

```diff
@@ -99,7 +101,7 @@ def set_edited_row_checkbox_to_true(
     # ignore static type checking here,
     # see https://github.com/pandas-dev/pandas-stubs/issues/256
```

Comment:
```suggestion
```
This comment is now lying. We don't need to ignore static type-checking *here*.

But possibly the comment could go elsewhere.